backend/controllers/productController.js

```
@@ -0,0 +1,187 @@
+//       error: "Internal server error",
+//       message: "",
+//     });
+//   }
+// });
+
+// new api for pagenation
+const listProduct = asyncHandler(async (req, res) => {
+  try {
+    const { page = 1, limit = 20, category, subCategory, sort } = req.query;
+    const skip = (page - 1) * limit;
+    let filter = {};
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'filter' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)